src/loaders/compute_tools/tool_common.py

```diff
@@ -357,12 +358,15 @@ def _create_metadata_file(
     # original genome id and source genome file info.
 
     # create tool genome identifier metadata file
-    genome_meta_file_path = os.path.join(batch_dir, loader_common_names.GENOME_METADATA_FILE)
+    genome_meta_file = loader_common_names.GENOME_METADATA_FILE
+    genome_meta_file_path = os.path.join(batch_dir, genome_meta_file)
     with open(genome_meta_file_path, "w") as meta_file:
         for genome_id, genome_meta_info in meta.items():
             meta_file.write(
                 f'{genome_meta_info[_META_TOOL_IDENTIFIER]}\t{genome_id}\t'
-                + f'{genome_meta_info[_META_SOURCE_FILE]}\n'
+                + f'{genome_meta_info[_META_SOURCE_FILE]}\t'
+                + f'{genome_meta_info[_META_SOURCE_DIR]}\t'
+                + f'{genome_meta_file}\n'
```

Comment:
I think you misunderstood the task. It should include the metadata file created when assembly files were downloaded. i.e.  Acidovorax_sp_CF301.meta for Acidovorax_sp_CF301.fa. Then during the parsing step, this meta data can be used to retrieve the UPA.

Reply:
Yeah, now it makes more sense after [RE2022-160](https://kbase-jira.atlassian.net/jira/software/projects/RE2022/boards/78?selectedIssue=RE2022-160)

Updated!!